gr-blocks/lib/ConfigChecks.cmake

```diff
@@ -1,8 +1,8 @@
-# Copyright 2012 Free Software Foundation, Inc.
+#Copyright 2012 Free Software Foundation, Inc.
 #
-# This file is part of GNU Radio
+#This file is part of GNU Radio
 #
-# SPDX-License-Identifier: GPL-3.0-or-later
+#SPDX - License - Identifier : GPL - 3.0 - or -later
```

Comment:
Were these changes intentional?

Reply:
No, that was clang-format it seems which I let run on the CMake file to increase readability of the C++ code snippets

Reply:
I'm not a fan of these changes, but if we want them, maybe they should go onto their own commit. The comments look weird now, and I don't think the snippets required increased readability.